07-reactjs/flux-react/components/button.js

```diff
@@ -0,0 +1,23 @@
+var React = require('react');
+
+var ButtonComponent = React.createClass({
+
+  getButtonProps: function () {
+    return{
+        className: 'btn btn-default btn-sm',
+        type: 'submit'
+    }
+  },
+
+  render: function () {
+    return (
+      <button {...this.getButtonProps()}>
```

Comment:
if you are getting the main component props this method should be called like getProps instead